app-text/apvlv/apvlv-0.4.0.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit xdg cmake
+
+DESCRIPTION="apvlv is a PDF/EPUB Viewer and its behaviour like Vim"
+HOMEPAGE="https://github.com/naihe2010/apvlv/"
+SRC_URI="https://github.com/naihe2010/apvlv/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug djvu"
```

Comment:
These use flags aren't doing anything in this ebuild? Maybe you forgot to write the rest? :)